sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -833,6 +838,264 @@ def noop(table, **kwargs):
               test_client=client))
 
 
+class BigQueryStreamingInsertsErrorHandling(unittest.TestCase):
```

Comment:
I think this suite should be skipped if bigquery fails to import, like the other suites in this file. Unfortunately I think that even with that, this test will break without GCP dependencies, since the test is parameterized by objects in `google.api.core.exceptions`.

Reply:
ah makes sense I'll add that